tests/blazesym.rs

```diff
@@ -134,6 +134,49 @@ fn symbolize_elf_dwarf_gsym() {
     test(src, true);
 }
 
+/// Make sure that we report (enabled) or don't report (disabled) inlined
+/// functions with a Gsym source.
+#[test]
+fn symbolize_gsym_inlined() {
+    fn test(src: symbolize::Source, inlined_fns: bool) {
+        let symbolizer = Symbolizer::builder()
+            .enable_inlined_fns(inlined_fns)
+            .build();
+        let results = symbolizer
+            .symbolize(&src, &[0x200020a])
+            .unwrap()
+            .into_iter()
+            .collect::<Vec<_>>();
+        assert_eq!(results.len(), 1);
+
+        let (result, _addr_idx) = &results[0];
```

Comment:
can you please also check the non-inlined symbol name and code_info here, regardless of `inlined_fns`?

Reply:
Will do.